website/static/js/registrationUtils.js

```diff
@@ -779,7 +781,54 @@ RegistrationEditor.prototype.save = function() {
     }
     return true;
 };
+/**
+ * Returns the `user_fullname` of each contributor attached to a node.
+ **/
+RegistrationEditor.prototype.getContributors = function() {
+    var contributorsUrl = window.contextVars.node.urls.api + 'contributors_abbrev/';
+    var contributorNames = [];
+    $osf.ajaxJSON('get', contributorsUrl).done(function(data) {
+        $.each(data.contributors, function(idx, val) {
+            contributorNames.push(val.user_fullname);
+        });
+    });
+    return contributorNames;
+};
+/**
+ * Opens a bootbox dialog with a checkbox list of each contributor
+ * the user has the option to import all contributors or to select
+ * each one individually.
+ **/
+RegistrationEditor.prototype.authorDialog = function() {
+    var self = this;
 
+    bootbox.dialog({
+        title: 'Choose which contributors to import:',
+        message: function() {
+            ko.renderTemplate('importContributors', self, {}, this, 'replaceNode');
+        },
+        buttons: {
+            select: {
+                label: 'Import',
+                className: 'btn-primary pull-left',
+                callback: function() {
+                    var boxes = document.querySelectorAll('input[type="checkbox"]');
+                    var authorString = '';
+                    $.each(boxes, function(i, box) {
+                        if( this.checked ) {
+                            authorString += this.value + ', ';
```

Comment:
This leaves a trailing comma at the end after the last author name. (even if only one author is selected)

An alternative would be to get the authors list as an array, then use `authors_array.join( ', ')`